settings.gradle.kts

```diff
@@ -22,18 +22,45 @@ pluginManagement {
   // Cannot use a settings-script global variable/value, so pass the 'versions' Properties via
   // settings.extra around.
   val versions = java.util.Properties()
+  val pluginIdPattern =
+    java.util.regex.Pattern.compile("\\s*id\\(\"([^\"]+)\"\\) version \"([^\"]+)\"\\s*")
   settings.extra["nessieBuildTools.versions"] = versions
 
-  val versionIdeaExtPlugin = "1.1.4"
-  versions["versionIdeaExtPlugin"] = versionIdeaExtPlugin
-  val versionSpotlessPlugin = "6.7.0"
-  versions["versionSpotlessPlugin"] = versionSpotlessPlugin
-
   plugins {
-    id("com.gradle.plugin-publish") version "1.0.0-rc-2"
-    id("com.diffplug.spotless") version versionSpotlessPlugin
-    id("io.github.gradle-nexus.publish-plugin") version "1.1.0"
-    id("org.jetbrains.gradle.plugin.idea-ext") version versionIdeaExtPlugin
+
+    // Note: this is NOT a real project but a hack for dependabot to manage the plugin versions.
```

Comment:
referring to "this is NOT a real project" seems odd here...
shouldnt this comment be just more like:
`pull in plugin versions from dependabot dummy project` ?